django/cantusdb_project/main_app/tests/test_functions.py

```diff
@@ -77,7 +66,10 @@ def test_syllabify_word(self):
             ("triumphas", ["tri", "um", "phas"]),
             ("phylosophia", ["phy", "lo", "so", "phi", "a"]),
             # fl
-            ("flores", ["flo", "res"],),
+            (
+                "flores",
+                ["flo", "res"],
```

Comment:
This confused me for a while, but it appears that the trailing comma after `["flo", "res"]` caused black to split this onto multiple lines whereas the previous tuples were kept on one line. Working to standardize this currently.